src/filtering/filterInput.component.tsx

```diff
@@ -62,8 +62,74 @@ const FilterInput = (props: FilterInputProps) => {
         setError('');
         setInputIndex((prevIndex) => prevIndex + -1);
       }
+      // allow selecting operators, channels, strings
+      // and numbers with Space key down
+      if (e.key === ' ') {
+        const operator = operators.find((opt) => opt.value === inputValue);
```

Comment:
I wonder if it would be nice to have the same behaviour as the channel behaviour - if there's only 1 operator left as an option then space selects it. Useful for some operators like "not" where it saves you having to type the `o` and `t`

Reply:
I thought about that but the reason why i didn't implement it like that was because it wouldn't work for all operators such as "<"  because "<=" exists there would always be two values.

Reply:
ah true - I thought that there was a reason but I couldn't remember what. You could handle the cases separately I suppose, i.e. where operator matches exactly even if there's more than one, and if there's only one as well, but that may be a little overkill 😅